crates/llms/src/chat/mistral.rs

```diff
@@ -617,7 +619,11 @@ fn stream_from_response(
                     // Only reacable if `stream=false`.
                     unreachable!()
                 },
-            }
+                MistralResponse::Raw{..} => {
+                    // We set `return_raw_logits: false`
+                    unreachable!()
```

Comment:
Add the message to the macro itself, not as a comment - in case it ever happens, then the message will at least have why the assumption that was broken.